subprojects/composite-builds/src/main/java/org/gradle/composite/internal/IncludedBuildDependencyMetadataBuilder.java

```diff
@@ -77,8 +77,8 @@ private LocalComponentMetadata createCompositeCopy(IncludedBuild build, ProjectC
             compositeComponentMetadata.addConfiguration(configurationName,
                 originalConfiguration.getDescription(), originalConfiguration.getExtendsFrom(), originalConfiguration.getHierarchy(),
                 originalConfiguration.isVisible(), originalConfiguration.isTransitive(), originalConfiguration.getAttributes(),
-                originalConfiguration.isConsumeOrPublishAllowed(),
-                originalConfiguration.isQueryOrResolveAllowed());
+                originalConfiguration.isCanBeConsumed(),
```

Comment:
I just followed the conventions found in the Gradle codebase. I find it awkward too, but it looks nicer in Groovy:

```
config.canBeConsumed
```